src/i18n/en.go

```diff
@@ -273,6 +273,7 @@ more info: https://docs.zerops.io/documentation/cli/authorization.html`
 	GrpcVpnApiTimeout = "zerops vpn server didn't respond within assigned time, try it again later"
 
 	HintChangeRegion = "hint: try to change your region (you can list available regions using `zcli region list`)"
+	ContextDeadline  = "context should have deadline"
```

Comment:
as a user I have absolutely no idea what that means or what to do, can we suggest or describe it more precisely?

Reply:
this is more like error for dev, ping function expects context to has deadline 
```
	deadline, has := ctx.Deadline()
	if !has {
		return errors.New(i18n.ContextDeadline)
	}
```
any for better solution?

Reply:
I would return internal server error or something like that